cluster/gce/manifests/cloud-controller-manager.manifest

```diff
@@ -0,0 +1,113 @@
+{
+"apiVersion": "v1",
+"kind": "Pod",
+"metadata": {
+  "name":"cloud-controller-manager",
+  "namespace": "kube-system",
+  "labels": {
+    "tier": "control-plane",
+    "component": "cloud-controller-manager"
+  }
+},
+"spec":{
+"securityContext": {
+  {{runAsUser}}
+  {{runAsGroup}}
+  {{supplementalGroups}}
+  "seccompProfile": {
+      "type": "RuntimeDefault"
+  }
+},
+"priorityClass": "system-node-critical",
+"hostNetwork": true,
+"containers":[
+    {
+    "name": "cloud-controller-manager",
+    "image": "gcr.io/k8s-staging-cloud-provider-gcp/cloud-controller-manager:v1.25.2-alpha_ae91c1fc0c443c464a4c878ffa2a4544483c6d1f",
```

Comment:
Too bad we can't add comments in json, otherwise we should add TODO here to use an image based on a release tag

Reply:
What do we do about this?

Can we `sed` it?

Reply:
@cheftako and I will follow-up on this once we have images based on actual releases